### ISSUE TITLE
CKEditor is missing information for keyboard only users on how to access the toolbar

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov#570]
### Observed behaviour
When creating content, in the 'Body' field there is a formatting toolbar above, there are no instructions on how to access this tool bar for keyboard only users and for screen reader users they are told use key Alt+0 for help but are not told more specifically to access help on Accessibility instructions to access the toolbar, screen reader user doesn’t know that there is a toolbar until after they press Alt+0. Keyboard only user has no instructions on how to access the toolbar.

### Expected results
Keyboard only uses should see a message on screen telling them to press Alt+0 for instruction on how to access the toolbar, Screen Reader users should get the same message. All users should have access to the same information.

### WCAG 2.1 references
WCAG 2.1 4.1.2 Use standard controls, or ensure that non-standard controls are properly marked up using WAI-ARIA
WCAG 2.1 2.1.1 All functionality is operable with a keyboard.
